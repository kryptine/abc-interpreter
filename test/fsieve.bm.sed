s/3000/10000/
s/Start.*/Start = sum [Select [2,3:Primes] NrOfPrimes \\\\ _ <- [1..200]]/
