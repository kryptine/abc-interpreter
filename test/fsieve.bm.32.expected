20945800
halt at 4
471012 53276 524288
